README.md

```diff
@@ -1,79 +1,83 @@
-# terraform-google-event-function
+# Event Function
 
-This module configures a system which responds to filtered Stackdriver
-Logging events by invoking a Cloud Functions function.
+This module configures a system which responds to events by invoking a
+Cloud Functions function.
 
-A project-level Stackdriver Logging export uses a provided filter to
-identify events of interest and publish them to a dedicated Pub/Sub
-topic. A Cloud Functions function subscribes to the topic and uses
-provided source code to process each event. The source code is
-retrieved from an archive which is created locally and stored in a
-Storage bucket.
+The root module configures a function sourced from a directory on
+localhost to respond to a given event trigger. The source directory is
+compressed and uploaded as a Cloud Storage bucket object which will be
+leveraged by the function.
+
+Alternatively, the
+[function-sourced-from-repository submodule][f6n-sourced-from-r8y-s7e]
```

Comment:
This is a subjective opinion - the truncation here might be surprising if someone's not familiar with this style of abbreviation. It might be helpful to expand this or change the naming so we use more commonly used abbreviations - but of course this is just discussing the naming of anchors and is a non-functional concern.

Resolve this comment at your discretion.